lib/functions-custom.php

```diff
@@ -148,18 +148,18 @@ function front_page_posts() {
 
 
 // Render related posts section from artist array
-function render_related_by_artist($artist_slug_array) {
+function render_related_by_artist($current_post_id) {
```

Comment:
function name is a bit confusing as this appears to be able to deal with more than one artist but the function is singular